### DISCUSSION TITLE
Possible bug in semicolon-seperated expressions?

### DISCUSSION THREAD

**Discussion by serkor1** [posit-dev/positron#5240] [Q&A] [answered]
Hi,

If I use RStudio I can do the following,

```R
a <- 10; b <- 10
a + b
```

But if I do the same in Positron, it will give the following error,

```R
Error:
! object 'b' not found
```

However, if I run the following code in Positron

```R
{a <- 10; b <- 10}
a + b
```

All expressions gets evaluated correctly. Is this working as intended or is it a bug?

**Comment by TimTaylor** [comment, marked as answer]
Looks like this has been raised as a bug over in https://github.com/posit-dev/positron/issues/4317.

**Reply by serkor1** [reply]
Yup. I need to refine my GitHub searching skills. Sorry. 🤦‍♀️ 

Edit: I searched through issues with "semicolon" and not "semi-colon", now that I think of it...

**Reply by lionel-** [reply]
I've prioritised this issue for the December release.
